app/views/shared/_modal_state_change.html.haml

```diff
@@ -1,4 +1,5 @@
 - p1 ||= nil
+- p2 ||= nil
```

Comment:
What is p1 and p2 supposed to represent?
Can we come up with more descriptive names?